p5js/FaceApi/FaceApi_Video_Landmarks_LocalModels/sketch.js

```diff
@@ -0,0 +1,124 @@
+let faceapi;
+let video;
+let detections;
+
+// relative path to your models from window.location.pathname
+const options = {
+    withFaceLandmarks: true,
+    withFaceExpressions: false,
+    withFaceDescriptors: false,
+    Mobilenetv1Model: 'models',
```

Comment:
Are these almost always going to be the same? Maybe something like `modelPath: path` would be simpler?

Reply:
@shiffman - actually, in this case, by default we read in the data via URL. If people will be specifying a directory path to the varying models, it could be that people have different directories for different models. If we only have "modelPath", then we force people to always use 1 directory. I'd keep it a bit more open ended since almost 99% of the time people will use the pretrained models that are being read in via URL?

Reply:
yes, that makes sense!!